apps/sms/views/conversation/js/recipients.js

```diff
@@ -431,7 +431,7 @@
     new GestureDetector(outer).startDetecting();
 
     // Set focus on the last "placeholder" element
-    this.reset().focus();
+    this.reset();
```

Comment:
I don't know why this was there -- I don't see the point of focusing this at instanciation time. So I removed it but if you can know why it was here I can add it back.